### ISSUE TITLE
Link to forked instance

### ISSUE THREAD

**Issue by donvadicastro** [angular-fullstack/generator-angular-fullstack#1494]
Hello. I use forked generator verison and trying to test my changes locally. For this purpose I'm using
```
npm link
```
to map to local generator. "Linking" process works fine, but when I'm trying to run I'm receiving next error
![image](https://cloud.githubusercontent.com/assets/8946681/11776922/72296144-a253-11e5-863a-253aaf3f0675.png)

What I'm doing wrong? Thanks.

**Comment by Awk34** [comment]
You should not be using Babel on your `node_modules` folder. I would guess that your entry point ([`app/index.js`](https://github.com/angular-fullstack/generator-angular-fullstack/blob/master/app/index.js#L4-L6)) is not excluding the `node_modules` folder for transpilation.

**Comment by donvadicastro** [comment]
Actually I found out another way to use forked solution without any transformation. I've analyzed whats going when "npm link" occurs - on windows machine it creates virtual reference to your project folder inside npm local repository folder.

This is not actually copy of your data - its is just like a shortcut to your repository in different place. When I tried remove this link and copy my data to this location - all this staff starts working as expected.

Using this experience I'm going to clone my git solution directly to npm local repository destination while developing and debugging.
